requirements/dev.txt

```diff
@@ -116,7 +112,7 @@ pep517==0.12.0
     # via
     #   -r requirements/pip-tools.txt
     #   pip-tools
-pillow==8.4.0
+pillow==9.0.0
```

Comment:
There isn't any breaking change mentioned in the `pillow==9.0.0` [CHANGELOG]( https://github.com/python-pillow/Pillow/blob/main/CHANGES.rst#900-2022-01-02) so it should be good to merge.